addons/uis.md

```diff
@@ -11,17 +11,18 @@ openHAB offers a variety of frontends for user interaction, from administrative
 
 | Web Application | Description   |
 |-----------------|---------------|
-| [Basic UI]({{base}}/addons/uis/basic/readme.html) | The Basic UI is an HTML5 web application in Material Design, designed for operating openHAB. |
-| [Classic UI]({{base}}/addons/uis/classic/readme.html) | The Classic UI is the original openHAB 1.x webui, designed for operating openHAB. |
-| [HABmin]({{base}}/addons/uis/habmin/readme.html) | HABmin is a modern, professional and portable user interface for openHAB, providing both user and administrative functions. |
-| [HABPanel]({{base}}/addons/uis/habpanel/readme.html) | HABPanel is a lightweight dashboard interface for openHAB. |
-| [Paper UI]({{base}}/addons/uis/paper/readme.html) | The Paper UI is an AngularJS-based HTML5 web application in Material Design, designed for setup and administration purposes. |
+| [Basic UI]({{base}}/configuration/ui/basic/) | The Basic UI is an HTML5 web application in Material Design, designed for operating openHAB. |
+| [Classic UI]({{base}}/configuration/ui/classic/) | The Classic UI is the original openHAB 1.x webui, designed for operating openHAB. |
+| [HABmin]({{base}}/configuration/ui/habmin/l) | HABmin is a modern, professional and portable user interface for openHAB, providing both user and administrative functions. |
+| [HABPanel]({{base}}/configuration/ui/habpanel/) | HABPanel is a lightweight dashboard interface for openHAB. |
+| [Paper UI]({{base}}/configuration/ui/paperui/) | The Paper UI is an AngularJS-based HTML5 web application in Material Design, designed for setup and administration purposes. |
+| [HaBot]({{base}}/configuration/ui/habot/) | A chatbot for openHAB |
 
 | App     | Description          |
 |---------|----------------------|
-| [Android App]({{base}}/addons/uis/apps/android.html) | The native Android app to access openHAB on the go. |
-| [iOS App]({{base}}/addons/uis/apps/ios.html) | The native iOS app to access openHAB on the go. |
-| [Windows 10 App]({{base}}/addons/uis/apps/windows.html) | The native Windows 10 app to access openHAB on the go. |
+| [Android App]({{base}}/apps/android.html) | The native Android app to access openHAB on the go. |
```

Comment:
I think this one won't work since the source file is in the addons tree.

`/docs/apps/android.html` (without `{{base}}`) should do the trick.

Smae goes for the next two lines.

Reply:
You are absolutely correct the links above do not work as they were. Fixed links for all three apps. For consistency, I left it with {{base}} as I tried to use this with all other links to the docs too. If this is not longer necessary or desired, let me know and I can take all of them out.